extension/lib/settings_wrapper.js

```diff
@@ -13,7 +13,7 @@ if (window[namespace] === true) {
 const settingKey = {
 	SPEED: "speed", // WPM
 	TRACKER_CUSTOM: "tracker_custom",
-	TERM_DOCUMENT_FREQ: "termDocumentFreq",
+	DOCUMENT_FREQ: "termDocumentFreq",
```

Comment:
Changed names to documentFrequency instead of termDocumentFrequency to avoid confusion. EXCEPT for the settingKey value "termDocumentFreq", bc that was the value for my current local storage. Will change when I vaporize mine after testing to start from scratch